dist/mmcif_pdbx_v50.dic

```diff
@@ -142838,7 +142838,9 @@ save__reflns.gphl_percent_possible_ellipsoidal_obs_anomalous
 
 save__reflns.gphl_redundancy_anomalous
     _item_description.description
-;              GPhL TBD
+;              The overall redundancy of anomalous data, i.e. where
```

Comment:
Would it be slightly clearer if we said ""...anomalous data, i.e. data for which ..." (instead of "..anomaous data, i.e. where ..."?